### ISSUE TITLE
Revision -- Feedback 1

### ISSUE THREAD

**Issue by apoorvagnihotri** [distillpub/post--bayesian-optimization#3]
- [x] Add reference to matern kernel
- [x] For prior model figure, also add the uncertainty in the legend
- [x] make the alpha higher for the uncertainty in all the plots.
- [x] the first plot showing only the GT has thin lines, others have thicker lines. I would suggest having line width something in between the two.
- [x] Make all the plots uniform
- [x] Verify the header levels in HTML
- [x] Change the trivial acquisition function's name in plots
- [x] Slow down all the animations
- [x] Show the CDF in a darker (lesser alpha) so that we can clearly see what is happening.
- [x] Put hyper parameters in new line in the plots (after iteration number). Put them with mathematical symbols (latex in matplotlib). Also, no need to mention `HPar,ams` word in the plots. 
- [x] Do the same in html also. Like, do not write eps instead write <d-math>\epsilon</d-math>. I have made these corrections before EI. Please do it for everything that follows.
- [x] For Expected improvement, first write the formula in terms of expectation in improvement See slide labelled: An expected utility criterion in Nando De Freitas’ lecture: https://www.cs.ubc.ca/~nando/540-2013/lectures/l7.pdf
- [x] In the plots replace EI with \alpha_{EI} etc. for all the approaches
- [x] Write the formulae for GP-UCB and mention in text that it is similar to the first acquisition function that we had introduced. The text for GP-UCB seems broken (v to 3 …)
- [x] Don’t bring about Bernoulli discussion for Thomson sampling. Instead, add missing intuition: why choosing random samples from posterior and maximizing them will likely do a tradeoff between exploration and exploitation.
- [x] For hyperparams v/s params - just mention equation of Ridge regression and mention the params (\theta) and the hyperparams (\delta^2, learning rate,..)
- [x] The fonts on SVM and below plots are very small. Also, try with some other cmap options also. Also, the maximum point isn’t clearly visible for many of the plots. 

---

- [x] I think we may want to ensure that the convention for bayesian optimisation is the same across the method. We could stick to the convention you used in the EI section just now. I had written slightly differently for PI. It may be best to just use Nando de Freitas' convention as used in the paper linked in #2
- [x] https://arxiv.org/pdf/1012.2599.pdf is a nice paper to refer. I think we can just rename our first function as: UCB instead of ACQ1. You can also view Nando's lecture at 1:06 mins to understand why UCB would work.
- [ ] From paper mentioned in #2, we can update some text about GP-UCB also.
- [x] Don't write Plotting Posterior etc. as the title. Posterior should be enough.
- [x] For the PI visualisation wrt epsilon, I don't think the effect of epsilon is visible. We should be able to somehow see the larger variance points being chosen. Maybe plot the PI on a log-scale for the same. Also, make the epsilon = 0 as 0.0. This will ensure the plot doesn't jump.
- [x] For NNs, you might want to store all the results. So that if we have to just tweak the visualistions, we do not have to re-run the entire code.
- [x] The EI plots have PI as the ylabel currently.
- [x] For the random plot in the comparison section for 1d, run the random acquisition with multiple random initialisations and show the mean and the variance.

----
Need inputs
~~Add thompson sampling to SVM and others & NN for diff methods~~
